src/atomate2/vasp/sets/base.py

```diff
@@ -231,9 +231,17 @@ class VaspInputGenerator(InputGenerator):
         Functional to use. Default is to use the functional in the config dictionary.
         Valid values: "PBE", "PBE_52", "PBE_54", "LDA", "LDA_52", "LDA_54", "PW91",
         "LDA_US", "PW91_US".
-    auto_kspacing
-        Whether to set the kspacing value based on the band gap. Note, this only works
+    auto_metal_kpoints
+        If true and the system is metallic, try and use ``reciprocal_density_metal``
+        instead of ``reciprocal_density`` for metallic systems. Note, this only works
```

Comment:
Would it be beneficial to have an easy way to turn on the tighter k-spacing (e.g., when you already suspect you have a metal)? Or should the user then simply set the values themselves?

Reply:
Personally, I think then it's ok if the user just sets in themselves. If you already know it's a metal then I don't think there is much difference between setting the reciprocal density manually vs having another flag to set.

Reply:
The rest looks fine to me. I haven't have done a test run for phonons with it yet. I could do it later if you like.

Reply:
Thanks @JaGeo. I just tried running the phonon workflow and it seems to run fine.

Reply:
Great and thanks!

Reply:
I'm also onboard with the user-supplied metal kpoints thing since this seems more like a question about validating data after the fact so it should be taken care of in emmet.